c++ training/SteamProject/Videogame/Steam.cpp

```diff
@@ -0,0 +1,434 @@
+#include "Steam.h"
+//---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+void FSteam::OpenMainMenu()
+{
+	system("cls");
+
+	const std::string MainMenuMessage = "Welcome to the Steam store! What would you like to do? \n1 - Add a game\n2 - Modify or create game categories \n3 - View all games \n4 - Exit \nTELL ME: ";
+	std::cout << MainMenuMessage;
+
+	enum class MainMenuCommand { AddGameMenu = 1, CategoryManagerMenu, GameDisplayerMenu, Exit };
+	MainMenuCommand UserChoice;
+
+	float TemporalUserChoice;
```

Comment:
what do you mean about `TemporalUserChoice`?
what do you think about `UserChoiceMenu` or something seems like?

Reply:
it's temporal since it is likely to change, I'm dealing with floats because I intend to avoid cases such as 2.1 or other floats or doubles being recognized as the integer 2. So at this point it is just a possibility that variable's gonna be the one the app actually decides to run with. Hence the Temporal in the name.